src/App.tsx

```diff
@@ -15,36 +16,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
```

Comment:
As I understand it, in `README` they are asking to add a default sorting option to enam:

`Use enum SortType instead of string literals for possible sorting options (including the default one)`